Sources_v3/StreamChatUI/ChatChannelList/ChatSwipeableListItemView.swift

```diff
@@ -68,6 +68,20 @@ open class ChatSwipeableListItemView<ExtraData: ExtraDataTypes>: View, UIConfigP
 
     // MARK: Gesture recognizer
 
+    override public func gestureRecognizerShouldBegin(_ gestureRecognizer: UIGestureRecognizer) -> Bool {
+        guard let recognizer = gestureRecognizer as? UIPanGestureRecognizer else {
+            return super.gestureRecognizerShouldBegin(gestureRecognizer)
+        }
+        return abs(recognizer.translation(in: self).x) > abs(recognizer.translation(in: self).y)
```

Comment:
nit 1: it would be nice to get translation just once and reuse it
nit 2: What's happening here? Can you add a comment explaining that? :trollface:

Reply:
Done :)